### PR TITLE
Step 15: デザインを当てよう

### PR DESCRIPTION
**Pull request by ShihaoXu0420**
## Requirements 

- Bootstrapを導入して、これまで作成したアプリにデザインを当てましょう
  - 【オプション要件】自分でCSSを書いてデザインする

### CODE REVIEW

File: myapp/app/views/tasks/edit.html.erb [hunk 1]
```diff
@@ -1,53 +1,62 @@
...
+      </div>
+
+      <div class='form-check'>
+        <%= form.radio_button :status, 'completed' %>
+        <%= form.label :status_completed, 'Completed', class: 'form-label' %>
+      </div>
+    </div>
+
+    <p>
+      <%= form.submit 'Submit Edit', class: 'btn btn-success' %>
```
Comment (rr-mdalauddin-alamin) [inline myapp/app/views/tasks/edit.html.erb]:
```suggestion
      <%= form.submit 'Update', class: 'btn btn-success' %>
```

Comment (rr-mdalauddin-alamin) [reply myapp/app/views/tasks/edit.html.erb]: May be "Update" is more suitable for edit form here?

**Review by rr-mdalauddin-alamin — COMMENTED** [review]
I think it would be better to add instructions to run your application for any specific branch in your Readme.md or in the PR description. like [this](https://github.com/Fablic/training/blob/m-alamin/myapp/README.md)

so the reviewer can also clone and run the application easily. Its a practice we follow regularly.

File: myapp/app/views/tasks/edit.html.erb (hunk 1 above)
Comment (ShihaoXu0420) [reply myapp/app/views/tasks/edit.html.erb]: Yes you're right...🙂‍↕️

**Comment by ShihaoXu0420** [comment]
> I think it would be better to add instructions to run your application for any specific branch in your Readme.md or in the PR description. like [this](https://github.com/Fablic/training/blob/m-alamin/myapp/README.md)
> 
> so the reviewer can also clone and run the application easily. Its a practice we follow regularly.

I added an instruction doc and am going to link it to following PRs' descriptions.
caeb6dd

File: myapp/doc/Instructions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+### How to run rspec tests
+- To run rspec system tests
+  ```sh
+  docker compose exec api bundle exec rspec spec/system/ -fd
+  ```
+- To run rspec model tests
+  ```sh
+  docker compose exec api bundle exec rspec spec/system/ -fd
+  ```
```
Comment (rr-mdalauddin-alamin) [inline myapp/doc/Instructions.md]: both command are same?

File: myapp/spec/system/tasks_spec.rb [hunk 3]
```diff
@@ -234,7 +234,7 @@
       expect(page).to have_selector("input[type='radio'][value='in_progress']")
       expect(page).to have_selector("input[type='radio'][value='completed']")
 
-      expect(page).to have_button('Submit Task')
+      expect(page).to have_button('Create Task')
```
Comment (rr-mdalauddin-alamin) [inline myapp/spec/system/tasks_spec.rb]: May be only "Create" is more than okay here.
```suggestion
      expect(page).to have_button('Create')
```

**Review by rr-mdalauddin-alamin — COMMENTED** [review]
Otherwise LGTM for me

File: myapp/spec/system/tasks_spec.rb (hunk 3 above)
Comment (ShihaoXu0420) [reply myapp/spec/system/tasks_spec.rb]: Fixed it at 726d7d4

File: myapp/doc/Instructions.md (hunk 2 above)
Comment (ShihaoXu0420) [reply myapp/doc/Instructions.md]: Fixed it at 726d7d4

**Review by rr-mdalauddin-alamin — APPROVED** [review]
Thanks for fixing

LGTM

**Pull request merged**